app/mailers/notification_mailer.rb

```diff
@@ -1,11 +1,9 @@
 class NotificationMailer < ActionMailer::Base
-  default to: Proc.new { User.pluck(:email) },
-    from: "noreply@firehose-defcon.com"
- 
+  default to: Proc.new { User.where(post_notification: true).pluck(:email) },
```

Comment:
Small nitpick here, but if the email goes out `to` everyone, I believe it will cause everyone to see each others email addresses if they look in the headers.

Instead I think we should loop through each of the users and send an email one at a time.  Thoughts?  Besides that, this looks good.